src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -240,68 +241,82 @@ const ContentListTable = () => {
   } = data;
 
   const rowActions = useCallback(
-    (rowData: ContentItem): IAction[] =>
-      isRedHatRepository
-        ? features?.snapshots?.accessible
-          ? [
+    (rowData: ContentItem): IAction[] => {
+      const triggerSnapshotAction = {
+        isDisabled: actionTakingPlace || rowData?.status === 'Retrying',
+        title: 'Trigger Snapshot',
+        onClick: async () => {
+          try {
+            await axios.post(`/api/snapshot/${rowData?.uuid}/trigger`);
+          } catch (error) {
+            console.error('Error triggering snapshot:', error);
+          }
+        },
+      };
```

Comment:
As i mentioned before, you really need to look at https://github.com/content-services/content-sources-frontend/blob/main/src/services/Content/ContentApi.ts to see how we call apis from the frontend.

Reply:
now that you've got the api call in src/services/Content/ContentApi.ts   you need to call that from here instead of also trying to make the api call

Reply:
and for now i'd just call it directly.  We can iterate once you've got that working to use a mutation

Reply:
you're still callling the api in this file, as i mentioned before you need to call it from the ContentApi.ts file

Reply:
remember to call triggerSnapshot instead of calling the api here.

Reply:
you're not calling  triggerSnapshot() properly here, and notice that the api is being called here AND in the triggerSnapshot function.  There's no reason for that.  You need to remove the post() call here and just call triggerSnapshot.